### ISSUE TITLE
Re-running of tests don't work

### ISSUE THREAD

**Issue by jhnns** [peerigon/nof5#9]
When I use nof5, the tests aren't re-ran when pressing save. It only works when I create a new file. This occurs in windows and in Linux when using a virtual machine on windows.

**Comment by topa** [comment]
You mean this occurs with a Linux-VM on a Windows-Host?

**Comment by jhnns** [comment]
Yes. But the filesystem is not shared or something. It's fully hosted by Linux.

**Comment by topa** [comment]
So you're not working directly on the VM, instead you upload on change your files?

**Comment by jhnns** [comment]
Yes.

**Comment by topa** [comment]
I think the problem is caused by the environment you use. I suppose that the VM's filesystem is not emitting an event, that could be recognized by node.js. Maybe this will fixed in future, but I don't believe that I can fix it in anyway.
